java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/CreateCategoryController.java

```diff
@@ -20,7 +20,7 @@
 @RequestMapping("/v1/menu/{id}/category")
 public interface CreateCategoryController {
 
-  @PostMapping(consumes = "application/json", produces = "application/json")
+  @PostMapping(consumes = "application/json", produces = "application/json; charset=utf-8")
```

Comment:
What's the motivation behind all the charset values?

Reply:
the pact test expects "application/json; charset=utf-8".

Reply:
I think it's a pact issue.

UTF-8  is the default one for the rest controller.We don't need specify this in our controller but it looks like the client used by pact uses a ASCII charset if charset is not defined by the producer.